src/collections/_documentation/accounts/quotas.md

```diff
@@ -68,6 +68,21 @@ Not all events count towards your monthly event quota. In general, when events a
     content="Delete and discard and per-project rate limits are only available on Business and Enterprise plans"
 %}
 
+## Sampling
+
+In some situations, you may want to sample data within Sentry. Sampling is a valuable tool if you're concerned about the cost of Sentry, or if some data is naturally very high volume and noisy (like [_Security Policy Reports_]({%- link _documentation/error-reporting/security-policy-reporting.md -%})).
+
+All of Sentry's SDKs provide a `sampleRate` configuration. For example, in JavaScript:
```

Comment:
> All of Sentry's SDKs provide a `sampleRate` configuration.

I believe this is true of all unified SDK's. Is it true of all the legacy ones as well?

Reply:
I believe almost every SDK has some type of sampling but not 100% sure. For example, raven-JS has sample.

Reply:
Right, but it's "spelled" differently in some of the legacy SDK's, I believe. Worth checking to know for sure.